resources/sample_vaults/Tasks-Demo/Styling/Styling of Queries.md

```diff
@@ -0,0 +1,40 @@
+# Styling of Queries
+
+To test styling of queries, follow this:
```

Comment:
I understand why you've put this here, as it is to do with styling - but it's also to do with Manual Testing...
Tricky...

Reply:
I think on reflection I would put it in Manual Testing, as it is not useful for users at all - and so that better conveys its purpose.

Reply:
I think I misunderstood the structure of the demo vault, and some of your comments somewhat confuse me further :sweat_smile: Can you elaborate more on where this should be and how I should call it?

Reply:
It's fine where it is. I made the smoke-test file link to this new file. Resolving conversation.